docs/sample_config.yaml

```diff
@@ -2657,6 +2657,12 @@ opentracing:
 #
 #run_background_tasks_on: worker1
 
+# The worker that is used to run user directory update tasks
+# (e.g. users in public rooms, shared rooms between users.).
+#
+# If not provided, or null, this defaults to the main process.
+#update_user_directory_on: null
```

Comment:
this still doesn't match the conventions specified at https://github.com/matrix-org/synapse/blob/develop/docs/code_style.md#configuration-file-format. Please go and read it.

Reply:
I guess it's only fair to give you some clues.

> A line consisting of only `#`.
> ...
> For other options, the example should give some non-default value which is likely to be useful to the reader.

Reply:
Ah gotcha, will update. :+1: